curricula/urls.py

```diff
@@ -9,7 +9,7 @@
 
                        # Allow viewing concept documentations without the /documentation/ prefix
                        # so links work on both curriculumbuilder and docs.code.org
-                       url(r'^(?P<slug>/docs/concepts.*)/$', documentation_views.map_view, name="map_view"),
```

Comment:
Resetting this back. Was first changed by me here: https://github.com/code-dot-org/curriculumbuilder/commit/d527537eaf84ebc2c309fb9a57d60c3014c267c6 in attempt to change this 2 months ago.